24_clustering.Rmd

```diff
@@ -140,8 +108,7 @@ library(mia)
 library(vegan)
 
 # Load experimental data
-data(peerj13075)
-(tse <- peerj13075)
+tse <- peerj13075
```

Comment:
I would use the enterotype dataset as an example with all sample clustering examples.